src/codegen/utilities/function_helpers.py

```diff
@@ -11,22 +12,62 @@
     "CIGPS": "CI_GPS",
 }
 
+FUNCTION_RETURN_TYPE_MAP_SET = {
+    "char[]": "str",
+}
+
+# This custom regex list doesn't split the string before the number.
+CUSTOM_CAMEL_TO_SNAKE_CASE_REGEXES = [
```

Comment:
`INTERPRETER_IGNORED_FUNCTIONS` made me realize it would make sense to change this constant's prefix from `CUSTOM_` to `INTERPRETER_`.

Reply:
Updated.